package.json

```diff
@@ -29,7 +29,7 @@
 	"dependencies": {
 		"@sapphire/discord-utilities": "^2.0.1",
 		"@sapphire/discord.js-utilities": "^1.3.1",
-		"@sapphire/pieces": "^1.2.0",
+		"@sapphire/pieces": "sapphire-project/pieces#build",
```

Comment:
This should be reverted, or wait for @Favna to do a 1.2.1 release.

Reply:
Resolved in 1e48e602b1e7b439882a897a1a8b2775e8c3e0d5